src/SmartComponents/Recs/Details.js

```diff
@@ -165,7 +165,7 @@ const OverviewDetails = ({ match, fetchRuleAck, fetchTopics, fetchSystem, fetchR
     }, [fetchRuleAck, rule.reports_shown, rule.rule_id]);
 
     return <React.Fragment>
-        <Suspense fallback={<Loading />}> <TagsToolbar /> </Suspense>
+        {!isGlobalFilter() && <Suspense fallback={<Loading />}> <TagsToolbar /> </Suspense>}
```

Comment:
oh like this plan, many many thankyous for not blocking clean pushes from master to stable 🤤